packages/config/index.ts

```diff
@@ -122,7 +122,7 @@ export const RESET_TUTORIAL = location.search.includes('RESET_TUTORIAL')
 
 export const ENGINE_DEBUG_PANEL = location.search.includes('ENGINE_DEBUG_PANEL')
 export const SCENE_DEBUG_PANEL = location.search.includes('SCENE_DEBUG_PANEL') && !ENGINE_DEBUG_PANEL
-export const SHOW_FPS_COUNTER = location.search.includes('SHOW_FPS_COUNTER') || DEBUG
+export const SHOW_FPS_COUNTER = (location.search.includes('SHOW_FPS_COUNTER') || DEBUG) && WORLD_EXPLORER
```

Comment:
I think this may be a better condition, reverting the changes in `packages/unity-interface/dcl.ts` to respect the single responsibility of `SHOW_FPS_COUNTER`

```suggestion
export const SHOW_FPS_COUNTER = location.search.includes('SHOW_FPS_COUNTER') || SCENE_DEBUG_PANEL
```

Reply:
Is there some way to check which is the most used `param`?

Reply:
WDYT if we separate the EDITOR and PREVIEW booleans from the Debug section? With the latest configuration I pushed, we get the fps panel with DEBUG_MODE on; and it's not going to be dependant on the value of SCENE_DEBUG_PANEL.